### DISCUSSION TITLE
Having multiple schemas and using SQL databases results in failure.

### DISCUSSION THREAD

**Discussion by Sreenuraj** [radical-sh/public-docs#13] [Issues]
Getting "QueryFailedError: duplicate key value violates unique constraint "pg_type_typname_nsp_index"" when a project is having multiple schema. 
[npm.log](https://github.com/radical-sh/public-docs/files/14414997/npm.log)

error in sql:
`postgres-trial  | 2024-02-27 05:52:43.695 UTC [29] ERROR:  duplicate key value violates unique constraint "pg_type_typname_nsp_index"
postgres-trial  | 2024-02-27 05:52:43.695 UTC [29] DETAIL:  Key (typname, typnamespace)=(newtrials, 2200) already exists.
postgres-trial  | 2024-02-27 05:52:43.695 UTC [29] STATEMENT:  CREATE TABLE "newtrials" ("newTrailID" character varying NOT NULL, "newTrialName" character varying NOT NULL, CONSTRAINT "PK_125f911804df7fb74b0f649e8e0" PRIMARY KEY ("newTrailID"))`

**Comment by senthilsivanath** [comment]
this can be result of invalid migration or db container having a wrong state. 

Resolution:
Remove dist folder on the root of the project and rerun. 

Reference : https://stackoverflow.com/questions/67756722/receiving-queryfailederror-duplicate-key-value-violates-unique-constraint-pg
